packages/ws/src/Decorators/Broadcast.ts

```diff
@@ -0,0 +1,22 @@
+// broadcast (emit) message to all users on the same namespace (including the peer)
+
+import { initializeMetadata, initializeMetadataMethod } from '@vercube/core';
+import { BaseDecorator, createDecorator } from '@vercube/di';
+
+class BroadcastDecorator extends BaseDecorator {
```

Comment:
All decorators should be fully described by the `jsdoc` (you can look at `Get.ts` or other one from core package)